etc/org.opencastproject.index.service.impl.util.PublicationEventUpdateHandler.cfg

```diff
@@ -0,0 +1,14 @@
+# A comma separated list of publication channel identifiers to automatically republish to when saving to the asset
+# manager.
+# The available publication channel identifiers depends on your system configuration.
+# Common publication channels are:
+#   - engage-player: Standard publication for players
+#   - interal: Opencast internal publication (mainly for the editor)
+#   - api: for the external api
+# Default value: <empty>
+#publication.channel.ids =
+
+# A comma separated list of flavors of the metadata and ACLs to republish.
+# While you can in theory specify any flavor and thus republish anything you want, it is recommended to only specify
+# the flavors of your metadata catalogs and ACL files. Anything else may lead to unexpected results.
+publication.flavors = "dublincore/*, security/*"
```

Comment:
What happens if I leave this blank? :D

Also, are the quotes necessary?  My read of the code says that
`"dublincore/*, security/*"` becomes
```
[
  "dublincore/*",
  "security/*"
]
```
Which is probably not what we're looking for?  Unless I missed the quote-stripping code somewhere...

Reply:
> What happens if I leave this blank? :D

Then you publish all elements in the `""` flavor. Will fix that, thanks.

> Also, are the quotes necessary?

No, but it works with or without quotes (No, I can't point to the function that does this, sorry). It still makes sense to remove them for consistency across config options I suppose, so I'll do that.